src/_helpers.scss

```diff
@@ -433,6 +433,40 @@
 
 
 /**
+ * Hide elements for JavaScript users and non-JavaScript users, this is
+ * necessary when building accessible and progressively enhanced UI's. This
+ * will require functionality that can append a hook to an element
+ * (typically the `html` element) if JavaScript is on.
+ *
+ * N.B. it is okay to use `!important` here as we're doing it preemptively
+ * i.e. you know you will always want the rule it's applied too to take
+ * precedence.
```

Comment:
Be good to drop this comment as it's not being said with any of the other *many* helpers that use `!important`, if anywhere probably best in the intro block.